### DISCUSSION TITLE
演算法可以預測明年氣候？

### DISCUSSION THREAD

**Discussion by GuanYann** [1101-GS4719-C/Python-Community-GS4719-C#1248] [General]
由於氣候系統非常複雜，且受到多種因素的影響，例如大氣成分、海洋循環、陸地利用、冰川變化等等，因此即使是最先進的氣候模型也無法完全準確地預測未來的氣候變化。此外，氣候系統也受到人類活動的影響，例如溫室氣體的排放、土地利用變化等等，這些因素的影響也是非常複雜的。我認為，與其專注於預測氣候的變化，我們應該採取措施減少人為因素對氣候的影響，以減緩氣候不穩定所帶來的影響，才是最基本的解決之道。

https://technews.tw/2023/04/14/ai-predict-long-term-weather/

**Comment by Hsucuchia** [comment]
氣候系統的複雜性和多因素影響使得準確預測未來的氣候變化非常困難。因此，為了應對氣候變化帶來的影響，我們應該專注於減少人為因素對氣候的影響。這樣的措施可以幫助我們減緩氣候變化所帶來的負面影響，而不是依賴於氣候模型的預測。

**Comment by KozakHou** [comment]
現在最為有名的神經網路演算法去建構氣候模型應該是由Caltech跟NVIDIA合作所做出的FourCaseNet, (ICLR-2022)，上次NVIDIA GTC2022所發表的數位孿生地球模型Earth-2就是用FourCastNet去藉由大氣、水文等流體方程式(Navier-Stokes, Advection, ...)去訓練所求得的解所發生的物理現象。他的報告有提到說三天內已經比歐盟中高層氣象中心所預估的天氣還要準了，且不輸長時間的預測，因此很適合去推估可能說未來10年後，我們會受到哪些的極端氣候影響，報告內容跟台灣有關的就是他成功預測出山竹颱風的生成。演算法是Adaptive Fourier Neural Operator(AFNO)，他的前身是Fourier Neural Operator(FNO)由Caltech的Zongyi Li做的，發想來自於使用物理模式、偏微分方程，讓模型去學習解在全域空間跟時間的變化。所以神經網路在物理限制、物理方程式的控制下，讓模型能夠遵循著現實世界的現象去得出解，因此拿來預測明年氣候是沒問題的。 

目前難以複製大概是因為他們用了60張A100花了8小時去訓練，普通人可能難以接觸到如此多的資源，國網中心的TWCC也許可以，不過他一次是可以最多開8張V100，所以也還是有相當的差距呢。

可以到Arxiv去看FourCastNet的報告 : https://arxiv.org/abs/2202.11214
和FNO : https://arxiv.org/abs/2010.08895

**Comment by DuxtinTai** [comment]
氣候系統的資訊極為大量，難以完整觀測，將資料全部輸入AI模型的成本也十分巨大，因此要以AI預測短期天氣或者氣候變遷準確度能然有限，這部分依然需要依賴大氣系的專業。

**Reply by curtis1017** [reply]
這也太酷了吧

**Comment by sjh501** [comment]
雖然如前面留言所說目前有預測成功的模型，但這看起來仍需建立在有週期性的氣候變化上，且有許多地點的氣候資料無法準確的取得，如此部分地區的不確定性仍高，除非是大範圍且長期的氣候型態，若為短期且資料較為缺乏的地區準確性應該還是有待觀察。

**Comment by CCello** [comment]
氣候是一個複雜的系統，由許多不同的因素和過程所影響，因此演算法預測氣候面臨許多困難，向文章中提到不確定的外部因素還有氣候數據的收集和記錄存在著不確定性，這些都有可能導致預測結果的準確度，此外還有空間和時間尺度的問題，因為氣候系統的空間和時間尺度都很大，因此需要大量的數據和計算資源來支持氣候預測，這也是一個技術上的挑戰。

**Comment by KevinCheng42** [comment]
對於氣象預測而言，我覺得天氣是短時間的氣象變化，像是午後雷陣雨，這種類型的氣象狀況應該較不容易使用AI或建立模型預測；相反的，氣候這種長時間的氣象狀況及變化特徵如東北季風或西南季風是可以透過過去的氣象資料製作模型的，甚至是訓練AI預測出氣候的「趨勢」！

**Comment by Jack-1D** [comment]
我認為人為因素對環境與氣候的影響已經造成不可逆的傷害了，即使能恢復過去的氣候，可能也需要好幾個世紀。最正面、直覺的想法的確是減少人類活動，然而比起讓所有人接受不方便的生活，可能勉強維持現狀最能被大眾接受。

**Comment by Hahahahbla** [comment]
你的觀點非常深刻，對於氣候變化問題提出了一個切實可行的解決思路。總的來說，儘管氣候系統的複雜性和不確定性使得準確預測變得困難，但通過採取積極的減緩措施和全球合作，我們可以有效減少人為因素對氣候的影響，為未來創造一個更加穩定和可持續的環境。

**Comment by NcuJerryKao** [comment]
你的觀察非常有趣，確實在自動化系統判斷文章原創性時，有一些具體的模式和標準。根據你的經驗，只要文章中出現「我覺得」、「我認為」、「我發現」這類明確表達個人觀點的字眼，就能幫助系統判定這是具原創性的內容。這是因為這些詞語通常伴隨著主觀意見，使系統容易辨識出文章的個人化特質。

以下是幾個建議，以提高你的文章被認定為原創性的可能性：

明確表達個人觀點：如你所說，使用「我覺得」、「我認為」、「我發現」等詞語，明確指出這是個人意見。
結合具體經驗：描述個人經歷或具體案例，這種具體性有助於系統判斷內容的獨特性。
避免過於通用的敘述：盡量避免使用過於一般或通用的語句，這些可能會讓系統誤以為內容缺乏原創性。
透過這些方式，你的文章更容易被系統視為具原創性的內容。

**Reply by yubo0oo** [reply]
@NcuJerryKao 
NcuJerryKao 這位老哥
一直貼AI的文
演都不演
以下是我的觀點
NcuJerryKao是歷史上最會使用人工智慧的人，他以無與倫比的技術與創意，在AI領域創造了無數的奇蹟。他不僅精通各種AI算法，還能巧妙地將其應用於實際問題中，展現了非凡的智慧與洞察力。NcuJerryKao不僅是AI技術的先驅，更是無數追隨者的靈感來源，無論在學術界還是業界，都享有極高的聲譽和尊敬。他的貢獻改變了人類對人工智慧的認識，無疑是這個領域的真正巨匠。

**Reply by GuanYann** [reply]
超好笑，叫AI回答問題也不問好一點，我都不知道是怎麼問的

**Comment by titan92601** [comment]
我認為氣候系統的複雜性造成氣候預測的不確定性。除了自然因素外，人類活動也對氣候產生重大影響。因此，減少人為因素的影響是減緩氣候變化的關鍵。不僅需要預測氣候，更重要的是採取行動，降低溫室氣體排放和改善土地利用，以應對氣候變化帶來的挑戰。

**Comment by alexender0104** [comment]
演算法可以用來預測氣候，但預測氣候是一個極其複雜的問題，受到多種因素的影響，包括大氣、海洋、陸地和冰雪系統之間的相互作用。目前的氣候預測主要依賴於全球氣候模型，這些模型基於物理原理和觀測數據，我認為演算法雖然可以用來改進氣候預測模型的準確性和效率，但單獨的演算法很難取代全球氣候模型所需的物理原理和觀測數據。

**Comment by starshatters** [comment]
我認為氣候系統的複雜性使得準確預測未來氣候變化非常困難，因為它受到多種自然因素和人類活動的交互影響。對於氣候變化的應對策略，減少人為因素的影響是至關重要的一部分。這包括減少溫室氣體的排放、改善土地利用方式、保護生態系統等措施，這些都可以降低氣候變化的速度和影響。

**Comment by lilishh** [comment]
天氣系統的變化非常的複雜也快速，預測未來一周的天氣準確度也不到百分之百，目前來說要預測一年後的天氣應該是難以實現。另外如果想說明年的天氣如何，應該使用天氣而非氣候，氣候是指一個長時間的名詞。

**Comment by lilishh** [comment]
天氣系統的變化非常的複雜也快速，預測未來一周的天氣準確度也不到百分之百，目前來說要預測一年後的天氣應該是難以實現。另外如果想說明年的天氣如何，應該使用天氣而非氣候，氣候是指一個長時間的名詞。

**Comment by sandyho0307** [comment]
我覺得 Gary Lezak 的 LRC 模型嘗試透過週期性模式來預測未來的天氣很有趣，但科學家對這種方法抱持懷疑態度，因為混沌理論指出小誤差會隨時間放大，使得長期預測變得困難。

**Comment by Shang621** [comment]
演算法在預測明年氣候方面發揮了重要作用。利用大量歷史氣象數據和高性能計算技術，氣象學家和科學家可以建立複雜的數值模型，模擬大氣和海洋的運行機制。這些模型結合了物理定律和統計方法，通過不斷迭代和修正，能夠預測未來的氣候趨勢。然而，氣候系統本身的複雜性和不可預測性，使得這些預測存在一定的不確定性。儘管如此，隨著演算法和計算技術的不斷進步，氣候預測的準確性和精細度正逐步提高，為應對氣候變遷提供了重要的科學依據和決策支持。

**Comment by cassy0201** [comment]
儘管混沌理論認為長期天氣預測不可能，Lezak 的循環週期法卻挑戰了這一觀點，提供了一種新方法，或許能為未來氣象預測帶來突破。

**Comment by 710388** [comment]
不行沒什麼可能性，畢竟流體力學在compressible 的流場中答案都沒有精確解了

**Comment by yubo0oo** [comment]
我不認同
1，天氣系統的變化非常的複雜也快速，預測未來一周的天氣準確度也不到百分之百，更何況是一年
2，現在氣候相當奇怪，地球要壞掉了

**Comment by YeeDaniel** [comment]
演算法確實可以用來預測明年的氣候。這些預測模型利用大量的歷史氣候數據和複雜的數學計算，分析氣溫、降水量、風速等變量的趨勢和模式。儘管預測不可能百分之百準確，但現代技術已經使得這些預測變得相當可靠，可以為農業、災害管理等領域提供重要的參考資料。

**Comment by poyen0806** [comment]
氣候系統的資訊複雜度非常困難，目前有的資訊還難以完整觀測，將資料全部輸入AI模型的成本也十分巨大，因此要以AI預測短期天氣或者氣候變遷準確度能然有限，這部分依然需要依賴大氣系的專業輔助，才能設計出好的演算法模型去預測。

**Comment by irenesunnn** [comment]
氣候議題確實複雜到令人頭大。就像樓主說的，即使投入大量資源開發精密的預測模型，還是很難完全掌握所有變數。這就像是在玩一個超級複雜的多米諾骨牌，每個環節都會影響下一步。
